### ISSUE TITLE
Lint not working

### ISSUE THREAD

**Issue by LucasBassetti** [plexsystems/vscode-protolint#7]
The lint is not working in my vscode.

If I run `protolint lint` command in terminal it works fine.

---
vscode version: 1.39.2
macOS version: 10.15.1
---

I appreciate any help! Thanks!

**Comment by jpreese** [comment]
I'll take a look at this tonight @LucasBassetti. In the meantime, can you confirm which version of the extension you're using?

Also make sure that you have an extension that adds the proto or proto3 language to your Viscose. I personally use https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3

**Comment by LucasBassetti** [comment]
I'm using vscode-proto3 `0.3.0` and vscode-protolint `0.2.0`

**Comment by jpreese** [comment]
Thanks for the added information. I'm on vscode `1.37` and it seems to be working. Let me update to `1.39.2`

**Comment by jpreese** [comment]
Could you give some information about what your folder structure looks like for your project?

I'm mostly curious about what folder you're opening in `vscode` and where your `.protolint.yml` file is. I'm wondering if it's not able to find your configuration file in the setup you have.

**Comment by LucasBassetti** [comment]
```
root
  |_ protos
      example.proto
  .protolint.yaml
```

I also tried

```
root
   example.proto
  .protolint.yaml
```

**Comment by jpreese** [comment]
Hm.. Ok, everything at the surface level appears to be correct. Let me dig out my macbook and see if its OS related. 

You're running the actual `protolint` binary and I'm assuming the results are coming back as you expect. So your `protolint` version is correct.

The extension does need you to save the `.proto` file for it to kick in, so make sure you're attempting to make a change and save it. The results should be printed to the _problems_ window.

Is this your first time trying to use the extension? Or did this stop working after 0.2.0?

**Comment by LucasBassetti** [comment]
It's my first time.

Example of result with `protolint`:

![image](https://user-images.githubusercontent.com/1014326/68001024-19f3e980-fc41-11e9-9303-813fa6a9a51a.png)

**Comment by LucasBassetti** [comment]
I fixed it!

The problem is that GOPATH was not added to PATH by default. I added it using the commands below, but when I restarted the terminal protolint stop works and I need to run it again

```
export GOPATH=$HOME/go
export PATH=$PATH:$GOPATH/bin
```

I solved adding it as default using this [tutorial](https://www.cyberciti.biz/faq/appleosx-bash-unix-change-set-path-environment-variable/)

Thanks for helping me!
